bool state
uint64 origin_ns
uint64 last_ns
uint64 seq
